### ISSUE TITLE
Link to "Sequence Diagram of the Transcription Pipeline" on Transcripts README Broken

### ISSUE THREAD

**Issue by rejeses** [bitcointranscripts/bitcointranscripts#466]
Link to the sequence diagram of the transcription pipeline on Transcripts README broken on some devices.

- From my Windows computer, it's OK. (image 1)

![Image](https://github.com/bitcointranscripts/bitcointranscripts/assets/125397952/d24f38bc-0b35-4b01-92b2-742075ef1f46)

- From @Extheoisah MacOS, it's this way (image 2)

![Image](https://github.com/bitcointranscripts/bitcointranscripts/assets/125397952/210539ac-8841-47fc-8c30-45df85aefa64)

- From my Android mobile device, it's this way (image 3)

![Image](https://github.com/bitcointranscripts/bitcointranscripts/assets/125397952/0dce0082-3341-4535-acc0-f3afe022398a)

**Comment by kouloumos** [comment]
Good catch @rejeses. Initially I thought that this is an issue with rendering mermaiid diagrams on mobile phone, but it's not, as [this one works on mobile](https://gist.github.com/kouloumos/df23f38265c512a3cf687232a204a63c).

It has to do with that click-to-expand functionality that I have there. I am trying to figure out what the issue is. If not solved, I will just move the sequence diagram somewhere else.
